packages/node_modules/@ciscospark/widget-space/src/selector.js

```diff
@@ -5,23 +5,36 @@ const getWidget = (state) => state.widgetSpace;
 const getSpark = (state) => state.spark;
 const getMedia = (state) => state.media;
 const getConversation = (state) => state.conversation;
-const getUsers = (state) => state.users;
 const getCurrentUser = (state, ownProps) => ownProps.currentUser;
 const getFeatures = (state) => state.features;
 const getOwnProps = (state, ownProps) => ownProps;
 
+/**
+ * Get the other user in a 1:1 conversation.
+ * @param {conversation} conversation
+ * @param {participant} currentUser
+ * @returns {participant}
+ */
+export const getToUser = createSelector(
+  [getConversation, getCurrentUser],
+  /* eslint arrow-body-style: [0] */
+  (conversation, currentUser) => {
+    return conversation
```

Comment:
can we fix this style error and remove the eslint change?

Reply:
All the other functions are styled that way. I can refactor though.

Reply:
Refactored. The slight difference was the appearance of any lines of code above the `return` statement. If you have one, e.g., assignment, it's fine. If you have `return` first, it wants you to use the single line syntax.

Reply:
@mattnorris you should be able to do something like this:
``` js
(conversation, currentUser) => conversation
  .get('participants')
```
or 
``` js
(conversation, currentUser) =>
  conversation
    .get('participants')
```